docs/_plugins/data-providers/leaflet-node.md

```diff
@@ -0,0 +1,11 @@
+---
+name: Leaflet node
+category: Data Providers
+repo: [Gitlab](https://gitlab.com/Goldmaster/leaflet-node)
+author: Goldmaster
+author-url: https://GoldmastersITDM.co.uk
+demo: https://goldmaster.gitlab.io/leaflet-node
```

Comment:
Not working

Reply:
Hi there, I'm afraid I am unsure why. I did follow the guide on getting the demo to work as it runs on gitlab pages.

Despite following the correct info, the demo fails to show.

I do have a real world demo example linked as well.

I will do some further research on this issue.

Reply:
Have a check now, got the demo to work